### PR TITLE
Update priority_list.md

### PR DESCRIPTION
**Pull request by kloaf11**
Changes to the priority list. Including a tip for when using WLR. Special mention for SE and EoGS combo in mplus for ST rotation. As well as updated the misconception at the bottom for SE with LB to the current suggested stack amount. Also updated the date, expansion and patch number it's current for.

### CODE REVIEW

File: guide/general/priority_list.md [hunk 1]
```diff
@@ -99,9 +99,16 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
     - {{ site.data.spell.es }}
     - {{ site.data.spell.lb }}
 
+**Tip**
+When using { site.data.lengendary.windspeaker }}, {{ site.data.talent.sk }} and {{ site.data.talent.mote }} on single target. You can go {{ site.data.spell.lava_burst }}, {{ site.data.spell.lb }} (empowered by storm keeper), {{ site.data.spell.lava_burst }}, {{ site.data.spell.es }}, {{ site.data.spell.lava_burst }} (from WLR), {{ site.data.spell.lb }} (empowered by storm keeper)
```
Comment (Bloodmallet) [inline guide/general/priority_list.md:103]: Stormkeeper should be available as `{{ site.data.talent.stormkeeper }}`, too. Generally everything that references a spell, talent, or active effect like legendary should be a link.

Same goes for the "(from WLR)". Please make WLR a link, too

File: guide/general/priority_list.md [hunk 2]
```diff
@@ -99,9 +99,16 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
...
 
+#### {{ site.data.talent.se }} + {{ site.data.lengendary.eogs }}
+Combining this combo in mythic plus for single target. When you have lust up you drop hardcast lava burst from the rotation and replace with Lightning Bolt. Your new priority becomes {{ site.data.spell.earthquake }} with {{ site.data.lengendary.eogs }} buff, {{ site.data.spell.earth_shock }} when you don't have the {{ site.data.lengendary.eogs }} buff, {{ site.data.spell.lava_burst }} with {{ site.data.spell.lava_surge }}, {{ site.data.spell.lb }}. Note this only happens during this combo with lust up. Outside of that you maintain the normal Rotation until 18 stacks where you drop hard cast {{ site.data.spell.lava_burst }}. For Necrolords - you continue to consume {{ site.data.spell.primordial_wave }}'s buff when used on cooldown preferably with a {{ site.data.spell.lava_surge }} proc, but hardcasting a {{ site.data.spell.lava_burst }} is still worthwhile.
```
Comment (Bloodmallet) [inline guide/general/priority_list.md:110]: Please replace `lava burst`, `Lightning Bolt`, and `lust`.

**Review by Bloodmallet — CHANGES_REQUESTED** [review]

**Review by Bloodmallet — APPROVED** [review]
I didn't run it locally (no ruby installation available right now). Otherwise looks good.

File: guide/general/covenant.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ By pledging your allegiance to the Necrolords you will gain two new abilites:
 
 **{{site.data.spell.fleshcraft}}**
 
-{{site.data.spell.fleshcraft}} is a channeled ability that gives you a 20% HP shield for 2 minutes. This can be amplified to 50% of your HP if it gets cast while in the vicinity of a corpse.
+{{site.data.spell.fleshcraft}} is a channeled ability that gives you a 40% HP shield for 2 minutes. While chaannelling this reduces damage taken by 20%. Also passively reduces the cooldown by consuming nearby corpses. 1 second per corpse.
```
Comment (HawkCorrigan) [inline guide/general/covenant.md:43]: typo

File: guide/general/covenant.md [hunk 4]
```diff
@@ -100,4 +100,4 @@ By pledging your allegiance to the Venthyr you will gain two new abilites:
 {{site.data.spell.chain_harvest}}
 
 {{site.data.spell.chain_harvest}} is basically a special {{site.data.spell.cl}}. After a 1.5 second channel it deals damage to 5 targets and heals 5 friendlies. Its cooldown gets reduced by 5 seconds for every critical hit, including damage and heal.
-{{site.data.spell.lavish_harvest}} increases the critical hit chance of {{site.data.spell.chain_harvest}} by 10%.
\ No newline at end of file
+{{site.data.spell.lavish_harvest}} increases the critical hit chance of {{site.data.spell.chain_harvest}} by 10%.
```
Comment (HawkCorrigan) [inline guide/general/covenant.md:103]: either there is no change here or I am blind
doesnt matter either way

File: guide/general/faq.md [hunk 5]
```diff
@@ -44,13 +44,25 @@ Since the cooldown of {{ site.data.spell.meteor }} is the same as the duration o
...
 
 ### Q: "Do I use Lava Burst during Storm Elemental?"
 
 Yes. No. It depends. Check out the [Priority List]({{ site.baseurl }}{% link guide/general/priority_list.md %}) and special notes on that website for in depth information.
 
 ### Q: "What are some useful Macros?"
+ 
+### Q: "What is funnelling?"
+ 
+Funneling is the act of casting {{ site.data.spell.fs }} on multiple targets in order to generate {{ site.data.spell.ls }} procs with the intention of increasing your damage on one target.
```
Comment (HawkCorrigan) [inline guide/general/faq.md:57]: * on the main target

File: guide/general/faq.md [hunk 6]
```diff
@@ -44,13 +44,25 @@ Since the cooldown of {{ site.data.spell.meteor }} is the same as the duration o
...
+ 
+### Q: "How does {{ site.data.talent.ecs }} interact with proc to abilities?"
+ 
+ {{ site.data.talent.ecs }} does not interact with procs to abilities; {{ site.data.talent.ecs }} will only echo an ability at its base.
```
Comment (HawkCorrigan) [inline guide/general/faq.md:61]: I dont think this is understandable tbh. I also dont have too much of an idea of how to phrase it better

File: guide/general/faq.md [hunk 7]
```diff
@@ -44,13 +44,25 @@ Since the cooldown of {{ site.data.spell.meteor }} is the same as the duration o
...
+ 
+ ###Q: "Why is my {{ site.data.spell.ls }} proc no longer instant cast?
+ 
+ It's probably because you have the pvp talent {{ site.data.talent.col }}.
```
Comment (HawkCorrigan) [inline guide/general/faq.md:65]: should probably mention war mode

File: guide/general/faq.md [hunk 8]
```diff
@@ -84,3 +96,45 @@ You can also adapt this macro to dismiss the {{site.data.spell.ee}} after gettin
...
 /script PetDismiss()
 ```
+
+-T2 Macro - Swaps between {{ site.data.talent.ecs }} and {{ site.data.talent.eb }} depending on selected talent.
+```
+ #showtooltip
+/use [talent:2/2]Echoing Shock;[talent:2/3]Elemental Blast
+ ```
+ 
+ -T5 Macro - Swaps between  {{ site.data.talent.ag }} and {{ site.data.talent.wrt }} depending on selected talent. Casts {{ site.data.talent.wrt }} at your cursor.
```
Comment (HawkCorrigan) [inline guide/general/faq.md:106]: something went wrong here

File: guide/general/faq.md [hunk 9]
```diff
@@ -84,3 +96,45 @@ You can also adapt this macro to dismiss the {{site.data.spell.ee}} after gettin
...
+ ```
+ #showtooltip
+/use [talent:2/2]Echoing Shock;[talent:2/3]Elemental Blast
+ ```
+ 
+-T5 Macro - Swaps between :ancestral_guidance: Ancestral Guidance and :wind_rush_totem: Wind Rush Totem depending on selected talent. Casts :wind_rush_totem: Wind Rush Totem at your cursor.
```
Comment (HawkCorrigan) [inline guide/general/faq.md:112]: something went wrong from here on out, either this is a syntax I dont know or its not converted into the link format

File: guide/general/priority_list.md [hunk 10]
```diff
@@ -99,6 +99,18 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
     - {{ site.data.spell.es }}
     - {{ site.data.spell.lb }}
 
+#### {{ site.data.talent.se }} Single Target cases
+
+In aoe you only cast {{ site.data.spell.cl }} and {{ site.data.spell.eq }} at 3+ targets. In ST you want to {{ site.data.spell.fs }} before casting {{ site.data.talent.se }} and with {{ site.data.spell.bloodlust }} then you only cast {{ site.data.spell.lb }} and {{ site.data.spell.es }} with using {{ site.data.spell.fs }} and use {{ site.data.spell.ls }} proc for movement. Without {{ site.data.spell.bloodlust }} then you cast {{ site.data.spell.ls }} procs, {{ site.data.spell.lb }} and {{ site.data.spell.es }} at the highest level of gear pre 9.1. At lower levels of gear normal rotation is advised outside of lust and using {{ site.data.spell.ls }} procs during lust.
```
Comment (HawkCorrigan) [inline guide/general/priority_list.md:104]: This probably should be reworded.

File: guide/general/priority_list.md [hunk 11]
```diff
@@ -99,6 +99,18 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
...
+ 
+With {{ site.data.talent.eogs }} you still alternate between using {{ site.data.spell.es }} and {{ site.data.spell.eq }} to maintain the {{ site.data.talent.eogs }} buff.
+
+With {{ site.data.spell.primoridial wave }} you still cast it on cd. Favoring to use a {{ site.data.spell.ls }} but hardcasting {{ site.data.spell.lvb }} if needed to use the {{ site.data.spell.primoridial wave }} buff.
```
Comment (HawkCorrigan) [inline guide/general/priority_list.md:108]: the link is broken

File: guide/general/priority_list.md [hunk 12]
```diff
@@ -99,6 +99,18 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
...
+
+**Tip**
+When using {{ site.data.lengendary.windspeaker }}, {{ site.data.talent.sk }} and {{ site.data.talent.mote }} on single target. You can go {{ site.data.spell.lava_burst }}, {{ site.data.spell.lb }} (empowered by {{ site.data.talent.sk }}), {{ site.data.spell.lava_burst }}, {{ site.data.spell.es }}, {{ site.data.spell.lava_burst }} (from {{ site.data.lengendary.windspeaker }}), {{ site.data.spell.lb }} (empowered by {{ site.data.talent.sk }})
```
Comment (HawkCorrigan) [inline guide/general/priority_list.md:111]: isnt that in the opener anyway?
if it isn't it should rather be there than in the main rotational section

File: guide/general/priority_list.md [hunk 13]
```diff
@@ -128,7 +140,7 @@ Combining {{ site.data.talent.se }} with {{ site.data.talent.pe }} enables acces
...
+**Special mention** There are many caveats with {{ site.data.talent.mote }} when in AoE situations. *IF* you have two or more {{ site.data.azerite.igneous_potential }} equipped, it is generally worth using {{ site.data.spell.lava_surge }} procs to empower {{ site.data.spell.eq }}s, even on 4+ targets. On more than 4 targets, only apply flame shock to one of the targets, ideally a target with higher health than the rest. Use {{ site.data.spell.lava_surge }} procs on the afflicted target in order to empower {{ site.data.spell.eq }}. With this in mind, be careful not to overdo it; it will very likely result in a loss of DPS if enemies die before your {{ site.data.spell.eq }} finishes ticking when you could have used the spell earlier.
```
Comment (HawkCorrigan) [inline guide/general/priority_list.md:143]: this should just be removed and reworked completely

**Review by HawkCorrigan — CHANGES_REQUESTED** [review]

**Comment by HawkCorrigan** [comment]
grafik

for context

**Review by Bloodmallet — CHANGES_REQUESTED** [review]
I'm taking over. I grabbed your changes into a SE&L branch and will be working on that one. Thank you for the preparation. I'm just sad to see this one laying around for so long.

**Comment by kloaf11** [comment]
Thanks sorry about not being able to follow when it finally got feedback properly.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**